ntbs-service/Pages/Notifications/Edit/MDRDetails.cshtml

```diff
@@ -0,0 +1,123 @@
+                        var caseInUKGroupState = hasCaseInUKError ? Error : Standard;
+                    }
+                    <nhs-form-group nhs-form-group-type=@caseInUKGroupState aria-describedby="case-in-uk-error">
+                        <nhs-fieldset>
+                            <nhs-fieldset-legend nhs-legend-size="Standard">Was the contact a case in the UK?</nhs-fieldset-legend>
+                            <span id="case-in-uk-error" nhs-span-type="ErrorMessage"
+                                        asp-validation-for="MDRDetails.CaseInUKStatus" has-error="@hasCaseInUKError"></span>
+                            <div class="nhsuk-radios govuk-radios--conditional" data-module="govuk-radios">
+                                <div class="nhsuk-radios__item">
+                                    <input asp-for="MDRDetails.CaseInUKStatus" id="uk-yes" class="nhsuk-radios__input" type="radio" value="@Status.Yes" data-aria-controls="conditional-uk-conditional">
+                                    <label class="nhsuk-label nhsuk-radios__label" for="uk-yes">
+                                        Yes
+                                    </label>
+                                </div>
+                                <div class="govuk-radios__conditional govuk-radios__conditional--hidden" id="conditional-uk-conditional">
+                                    <validate-related-notification model="MDRDetails" inline-template>
+                                        @{
+                                            var hasRelatedNotificationError = !Model.IsValid("MDRDetails.RelatedNotificationId");
+                                            var relatedNotificationGroupState = hasRelatedNotificationError ? Error : Standard;
+                                        }
+                                        <nhs-form-group nhs-form-group-type=@relatedNotificationGroupState aria-describedby="related-notification-error">
+                                            <span nhs-span-type="ErrorMessage" ref="errorField" has-error="@hasRelatedNotificationError"
+                                                    asp-validation-for="MDRDetails.RelatedNotificationId" id="related-notification-error"></span>
+                                            <label nhs-label-type="Standard" asp-for="MDRDetails.RelatedNotificationId">
+                                                Contact's notification id
+                                            </label>
+                                            <input ref="inputField" v-on:blur="validate" is-error-input=@hasRelatedNotificationError nhs-input-type="Date"
+                                                    classes="nhsuk-input--width-10" asp-for="MDRDetails.RelatedNotificationId" type="number">
+                                            <notification-info v-if="isValid" v-bind:notification-info="relatedNotification"></notification-info>

```

Comment:
This won't display in javascript - will require a small refactor but should be achievable to preload the HTML to allow for this content to be rendered on 'get'

Reply:
Ah sorry that is deliberate - we're not showing this for non javascript users

Reply:
That makes total sense @jasiekmiko should we get a tech debt to roll out this logic to NHS number duplicates?

Currently this is implemented to prepopulate on load if duplicates are present.

Reply:
ah, interesting spot! It was specced differently, so perhaps we just ought to wait on user feedback for this :)